### PR TITLE
chore: Update Admin-WG Members

### PR DESCRIPTION
**Pull request by groundwater**
We generally seek consensus from other WGs. This has been 👍 by admin-wg.

Adding Deani (Slack Inc.) and Anton (Microsoft Developer Division) to Admin WG.

- [x] Releases
- [x] Upgrades
- [x] Outreach
- [x] C&S
- [ ] QA
- [ ] Ecosystem
- [x] Security
- [x] API

### CODE REVIEW

File: wg-administrative/README.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ It is important that other working groups have autonomy and agency to fulfill th
 | -------------------------------------------|----------------------|----------------------------| -------- |
 | <img src="https://github.com/groundwater.png" width=100 alt="@groundwater">  | Jacob Groundwater [@groundwater](https://github.com/groundwater) | Member | PT (San Francisco) |
 | <img src="https://github.com/mgc.png" width=100 alt="@mgc">  | Matt Crocker [@mgc](https://github.com/mgc) | **Chair** | PT (San Francisco) |
```
Comment (nornagon) [inline wg-administrative/README.md:22]: Was the idea to replace @mgc by @deanihansen here?

Comment (groundwater) [reply wg-administrative/README.md:22]: @mgc should still be there, adding @deanihansen was a previously approved decision to help balance the wg.

**Comment by jkleinsc** [comment]
Approved on behalf of API WG

**Comment by groundwater** [comment]
I merged assuming tacit approval of folks who haven't responded one way or the other.

**Pull request merged**